hydrogen-vm-render-script.js

```diff
@@ -39,53 +132,69 @@ function mountHydrogen() {
   //   urlCreator: {},
   //   timeline: {},
   // });
-  
+
   // tiles.subscribe({ onAdd: () => null, onUpdate: () => null });
   // console.log(tiles._tiles);
-  
+
   // for (const t of tiles) {
   //   console.log(t.displayName);
   // }
-  
-const roomData = {
-  name: "Room Foo",
-  id: "!abc:hs.tld"
-};
-const me = RoomMember.fromUserId(roomData.id, "@awef:cawef.com");
-const timeline = {
-  dispose() {},
-  entries: list,
-  async loadAtTop() { return true;},
-  me
-};
-const mediaRepository = new MediaRepository({
-  homeserver: "https://hs.tld",
-});
-const room = {
-  name: roomData.name,
-  id: roomData.id,
-  avatarUrl: roomData.avatarUrl,
-  avatarColorId: roomData.id,
-  mediaRepository: mediaRepository,
-};
-const timelineVM = new TimelineViewModel({
-  timeline,
-  tileOptions: {
-    tileClassForEntry,
+
+  // const roomData = {
+  //   name: 'Room Foo',
+  //   id: '!abc:hs.tld',
+  // };
+  // const me = RoomMember.fromUserId(roomData.id, '@awef:cawef.com');
+
+  // const timeline = {
+  //   dispose() {},
+  //   entries: list,
+  //   async loadAtTop() {
+  //     return true;
+  //   },
+  //   me,
+  // };
+
+  await client.sync.status.waitFor((s) => {
```

Comment:
perhaps worth to mention in a comment: this waits for the catchup (first) sync to finish. The request function will respond with an empty json object for any subsequent /sync requests. In short, this waits for the first sync with the data we want to inject to finish.